packages/web/src/shared/components/form/RichTextInput.tsx

```diff
@@ -66,7 +66,7 @@ const RichTextInput = forwardRef<RichTextInputRef, RichTextInputProps>(
     });
 
     useEffect(() => {
-      editor?.commands.setContent(value ?? '', false, {
+      editor?.commands.setContent(value ?? '', true, {
```

Comment:
Did you do additional testing to make sure this change doesn't break anything else? I wonder why we used `false` in the first place.

Reply:
We don't want to emit events when you update the internal state of the rich text editor when props change, only when the state changed through user input. That's why this is set to false.

I don't think we need the hidden input anymore, so I'd just modify the change event to return a string and the blur event to not have any arguments. That will just simplify things overall.